src/messages/generic_api.py

```diff
@@ -9,29 +9,47 @@ def __init__(self):
 
     def send_item(self, item: str, image:  np.ndarray, location: str):
         msg = f"Found {item} at {location}"
-        self._send(msg)
-        
+        title="no"
+        img="no"
```

Comment:
why we need title and img if the private _send() function does not utilize any of it?

Reply:
Before gleed redid the pr I made a bunch of comments on #430 about similar things just never copied them over here. Those should all be resolved before this is merged.

Reply:
> Before gleed redid the pr I made a bunch of comments on #430 about similar things just never copied them over here. Those should all be resolved before this is merged.

Oh the only comment I could see is the one from FranciscoZini or something about the gg club meme. You're still able to see your comments there?

Reply:
Yeah i wasn't worred about the meme that was fine haha you have to scroll down a bunch see them.  I can shift them to this pr.

Reply:
@egut125 sure. I don't see them still after checking again